data/json/game_balance.json

```diff
@@ -74,14 +74,14 @@
     "name": "PLAYER_HEALING_RATE",
     "info": "Set base player healing rate per turn. Default: 0.01",
```

Comment:
You still haven't updated the info text as mentioned in the earlyer comment. The same for the info text below.